modules/goalign/cleanseqs/meta.yml

```diff
@@ -0,0 +1,32 @@
+name: "goalign_cleanseqs"
+description: Filter out poor sequences from a multisequence alignment
+keywords:
+  - multiple sequence alignment
+  - quality control
+tools:
+  - "goalign":
+      description: "goalign is a set of command line tools to manipulate multiple alignments"
+      homepage: "https://github.com/evolbioinfo/goalign"
+      documentation: "https://github.com/evolbioinfo/goalign/blob/master/docs/index.md"
+      tool_dev_url: "https://github.com/evolbioinfo/goalign"
+      doi: "10.1093/nargab/lqab075"
+      licence: "['GPL v2']"
+
+input:
+  - aln:
+      type: file
+      description: MultiFASTA/Phylip alignment file
+      pattern: "*.{fasta,phy}"
```

Comment:
I'm not familiar with goalign, but should this be `*.{fasta,fas,fa,phy}`?

Reply:
The tools also supports compressed inputs, so it should account for those too